cellmlmanip/model.py

```diff
@@ -207,6 +208,8 @@ def get_equations_for(self, symbols, lexicographical_sort=True, recurse=True, st
         :param recurse: indicates whether to recurse the equation graph, or to return only the top level equations
         :param strip_units: if ``True``, all ``sympy.Dummy`` objects representing number with units will be replaced
             with ordinary sympy number objects.
+        :param: keep_unused_eqs indicates whether any unsued equations  should be kept. If keep_unused_eqs==False
```

Comment:
unsued => unused

Reply:
Problem is sympy simplifying out terms e.g. 0*x +y -> y which leads to the equation (for x) being there but the lhs not being used anywhere, see above for popper example.